pyproject.toml

```diff
@@ -53,9 +53,11 @@ keywords = [
   ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"  # Cannot remove upper-bound as isort needs >=3.6.1,<4.0
+python = ">=3.8.1,<4.0"  # Cannot remove upper-bound as isort needs >=3.6.1,<4.0
 Django = ">=2.2"
 django-crispy-forms = ">=1.13.0"
+syrupy = "^4.0.1"
+bs4 = "^0.0.1"
```

Comment:
**Issue:** Should these be in [`tool.poetry.dev-dependencies`](https://github.com/torchbox/tbxforms/pull/42/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R62)? To use `tbxforms` on a project, why do I need a snapshot plugin installed?

Reply:
also note that Poetry recommends using `[tool.poetry.group.dev.dependencies]` instead of `[tool.poetry.dev-dependencies]` since 1.2, the latter slated for deprecation